charts/cp-kafka/README.md

```diff
@@ -118,6 +118,7 @@ The configuration parameters in this section control the resources requested and
 | `imageTag` | Docker Image Tag of Confluent Kafka. | `5.5.0` |
 | `imagePullPolicy` | Docker Image Tag of Confluent Kafka. | `IfNotPresent` |
 | `imagePullSecrets` | Secrets to be used for private registries. | see [values.yaml](values.yaml) for details |
+| `enableEnterpriseFeatures` | If enterprise feautres shall be enabled. Requries the image to be `confluentinc/cp-enterprise-kafka`. If disabled, it works with `confluentinc/cp-kafka` | `true`
```

Comment:
You have a few typos here but maybe I would suggest something along those lines :
> If enterpriseFeatures flag is enabled, kafka image will be set to `confluentinc/cp-enterprise-kafka` otherwise it will set it to `confluentinc/cp-kafka` (Community version)

Reply:
@scoquelin sorry for the typo and the delay. Fixed the doc based on your suggestion.